trillian/ctfe/pem_cert_pool.go

```diff
@@ -91,7 +91,7 @@ func (p *PEMCertPool) AppendCertsFromPEMFile(pemFile string) error {
 	}
 
 	if !p.AppendCertsFromPEM(pemData) {
-		return fmt.Errorf("failed to parse PEM certs file: %v", err)
+		return fmt.Errorf("failed to parse PEM certs file")
```

Comment:
Good spot, thanks.

We normally use `errors.New()` when there is no format specifier/parameter though.